helm/end-user-ui/templates/ingress.yaml

```diff
@@ -21,8 +21,7 @@ spec:
   tls:
   - hosts:
     - "{{ .Release.Namespace }}.{{.Values.subdomain }}.{{ .Values.domain }}"
-  # We default to using the nginx self signed cert.
-  #   secretName: {{ printf "wildcard.%s.%s" .Release.Namespace .Values.domain }}
+    secretName: '{{ .Release.Namespace }}.{{.Values.subdomain }}.{{ .Values.domain }}'
```

Comment:
We want to make cert-manager optional. The default should be no cert-manager required.

Reply:
My impression was that if this secret name doesn't exist, it reverts to the same behavior as it would if the secretName wasn't specified. Is that sufficient?

Reply:
I believe If you specify a secretName that does not exist, the ingress will wait forever for that secret to be set before it can be provisioned. Have you tested this without cert-manager being installed, and does it still work?

Reply:
Testing indicates that when cert manager is not installed, and "certmanager.enabled" isn't set to true, everything works as it did prior to this PR.